src/state/translations-mixin.ts

```diff
@@ -119,7 +120,10 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
 
     protected hassConnected() {
       super.hassConnected();
-      getUserLocale(this.hass!).then((locale) => {
+
+      subscribeTranslationPreferences(this.hass!, async ({ value }) => {
```

Comment:
@bramkragten should I unsubscribe somewhere? Or does it reconnect automatically?

Reply:
Automatically